perfmetrics/scripts/continuous_test/ml_tests/tf/resnet/build.sh

```diff
@@ -1,4 +1,17 @@
 #!/bin/bash
```

Comment:
shouldn't the copy right be above /bin/bash ?. Same for other files also.

Reply:
The shebang needs to be at the top. More details in this [SO answer](https://stackoverflow.com/a/12911604/1654398)

Reply:
Thanks.